src/main/java/org/araymond/joal/core/bandwith/Speed.java

```diff
@@ -2,9 +2,11 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.NoArgsConstructor;
 
 @Data
 @AllArgsConstructor
+@NoArgsConstructor
 public class Speed {
-    private long bytesPerSeconds;
+    private long bytesPerSecond;
 }
```

Comment:
I don't quite like the use of @NoArgsConstructor.

I'd prefer to have a `public static final Speed ZERO = new Speed(0L);` in the Speed class and reference if with `Speed.ZERO` when needed